src/comms/usb_hid.cpp

```diff
@@ -51,46 +51,56 @@ void HIDLayer::ping() {
 		// attempt to read
 		if (read()) {
 			// if we read, attempt to write
-
-			// set the packet to be written's ID
-			m_outgoingPacket.set_id((uint16_t)m_packetsSent);
```

Comment:
No set id?

Reply:
Nah the set ID should happen manually, so it doesnt mess up custom packets